README.md

```diff
@@ -5,5 +5,6 @@ A simple script to download your fastest, accepted solutions from Leetcode!
 ## Usage
 
 ```shell
- python3 fetch_submissions.py <username>
+ pip install requests
```

Comment:
I think it is better to put it this way:

## Usage

If you do not have [requests](https://pypi.org/project/requests/2.7.0/) installed, first run `pip install requests`.
```python
 python3 fetch_submissions.py <username>
```

Reply:
Since [`pip install` caches the download requests ](https://pip.pypa.io/en/stable/reference/pip_install/#caching), I think it isn't necessary for the user to check if `requests` is installed. They can just run the `pip install requests` anyways and if they have already have it installed, the script terminates quickly and lets the user know that the requirement for `requests` and its dependencies is satisfied already.

Reply:
Yeah, I know that pip caches, I guess it is just a personal preference here because `requests` is way too commonly used for us to have it in the usage block I feel. There sure is no harm in running `pip install` once again but keeping the main code block for **Usage** simple is good too. And for the vast majority of people, `requests` should not be something new.